### PR TITLE
lshw test fix

### PR DESCRIPTION
**Pull request by PraveenPenguin**
1-Added tag for generic:lshw
added privileged tag for this test as lshw must be run as super user
it will only report partial information
2-Fixed generic:lshw
fixed all pep8 issue in script

### CODE REVIEW

File: generic/lshw.py [hunk 1]
```diff
@@ -28,9 +28,12 @@ class Lshwrun(Test):
     mainboard configuration, CPU version and speed,cache configuration, bus
     speed, etc. on DMI-capable x86 or IA-64 systems and on some PowerPC
     machines (PowerMac G4 is known to work).
+
+    :avocado: tags=privileged
     """
     interface = process.system_output("ip route show")
-    active_interface = process.system_output("ifconfig | head -1 | cut -d':' -f1", shell=True).strip().split()[0]
+    active_interface = process.system_output(
+        "ifconfig | head -1 | cut -d':' -f1", shell=True).strip().split()[0]
```
Comment (narasimhan-v) [inline generic/lshw.py]: @PraveenPenguin `ifconfig` will be deprecated, and `ip` command is preferred. Can you take care of that too here ?

File: generic/lshw.py [hunk 2]
```diff
@@ -161,9 +167,12 @@ def test_prod_id_serial(self):
         else:
             product_path = '/proc/device-tree/model'
             serial_path = '/proc/device-tree/system-id'
-        product_name = self.run_cmd_out("cat %s" % product_path).rstrip(' \t\r\n\0')
-        serial_num = self.run_cmd_out("cat %s" % serial_path).rstrip(' \t\r\n\0')
-        if product_name not in self.run_cmd_out("lshw | grep product | head -1"):
+        product_name = self.run_cmd_out(
+            "cat %s" % product_path).rstrip(' \t\r\n\0')
```
Comment (narasimhan-v) [inline generic/lshw.py]: can this be `genio.read_file()` ?

File: generic/lshw.py (hunk 1 above)
Comment (PraveenPenguin) [reply generic/lshw.py]: sure i will

File: generic/lshw.py (hunk 2 above)
Comment (PraveenPenguin) [reply generic/lshw.py]: ah yes i will do

**Comment by PraveenPenguin** [comment]
@narasimhan-v added the suggested change can you please have a look

**Pull request merged**